### ISSUE TITLE
Non-NWHC users getting notifications with changes to NWHC-only fields

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices#476]
**Describe the bug**
If a partner owns an event, and NWHC makes changes to the NWHC-only fields, the partner sees those field changes in their notifications, even though they can't see those fields on their event details page.

**Steps to reproduce the behavior**
1. abilenetest account has custom notifications for any updates to events in California and any updates to "Your Events"
2. abilenetest account created event 171009 in California on test site
3. logged in as jchipault (NWHC user) and edited the 5 NWHC-only fields
4. next day, abilenetest gets notification of changes to those 5 NWHC-only fields in both Custom and "Your Events" notifications

**Expected behavior**
Only users in NWHC should see changes to these 5 NWHC only fields in their notifications, since only these users can see these 5 fields on the event details page.

**Screenshots**
![image](https://user-images.githubusercontent.com/49287076/119355124-82168000-bc6a-11eb-8bcc-95adcd46212b.png)

**Comment by aaronstephenson** [comment]
@JChipault  Huh, that certainly wasn't intentional. I have a pretty good guess where the problem is (likely an oversight in the serializer field lists).

**Comment by aaronstephenson** [comment]
I think I found it. The code for these NHWC-only fields is just checking for event ownership/org membership. <slaps forehead>

```
if (cue_user.id == h.created_by.id or cue_user.organization.id == h.created_by.organization.id):
```

The fix is to change that line of code to check if cue_user is in NWHC. Quick fix.

**Comment by aaronstephenson** [comment]
Yep, that was the problem. After my fix, the output for user abilenetest is exactly as expected.

Here's the output from before the fix:

> Updates were made to the following sections (if applicable):
>
>Event quality check changed from None to 2021-05-21
>Event complete changed from False to True
>Event end date changed from None to 2021-05-21
>Event staff changed from None to Jennifer Chipault
>Event event status changed from New Event Record to Field Update Needed
>Event legal status changed from N/A to Legal-Closed
>Event legal number changed from "" to adfad
>An Event Diagnosis was created: Undetermined
>An Event Diagnosis was deleted: Pending
>Event Location end date changed from None to 2021-05-21

Here's the output from after the fix:

>Updates were made to the following sections (if applicable):
>
>Event complete changed from False to True
>Event end date changed from None to 2021-05-21
>An Event Diagnosis was created: Undetermined
>An Event Diagnosis was deleted: Pending
>Event Location end date changed from None to 2021-05-21

**Comment by aaronstephenson** [comment]
Fixed in 9406dfe4 and pushed to test server. @JChipault give this a test when you have the time.

**Comment by JChipault** [comment]
@aaronstephenson I'm signing off for the day. Will check this (and other tickets) more tomorrow.

**Comment by JChipault** [comment]
Set up some more testing, with notifications coming in over night, so will report back tomorrow.

**Comment by JChipault** [comment]
Hmmmm... not getting any notifications now for this scenario.

5/25
Event 171014 was created by JennyPartner1. Event is in California. JChipault logged in and added abilenetest as a read/write collaborator, changed NWHC-only fields, changed end date, and changed complete status.

5/26
Expected Custom notification (state=California) in abilenetest account that would include change to end date and complete status but NOT change to NWHC-only fields.
Abilenetest received notification about being added as a collaborator but no Custom notifications.

5/27
jchipault logged back into 171014 and changed NWHC-only fields, county, and end date.

Aaron did apache web server restart.

5/28
Abilenetest received no notifications. 

Since restart was done AFTER jchipault's changes on 5/27, jchipault did changes to 171014 again today and will check this weekend.

**Comment by aaronstephenson** [comment]
Oh for goodness sake, I made a boneheaded move. When I was debugging this problem, I set the `get_yesterday()` function to get 5 days prior (so I could grab event 171009 and test the fields being returned), and then forgot to change that back to 1 day prior.

I'll change it back to 1 day prior, and then rerun celerybeat so that the proper notifications get triggered.

**Comment by aaronstephenson** [comment]
Fixed in 23771b4 and pushed to test environment. Apache, celery, and celerybeat all restarted.

@JChipault note that the 'modified_date' on Event 171014 is May 28, so that won't appear until tomorrow's batch of notifications.

**Comment by JChipault** [comment]
Cool. What I received in the abilenetest account on May 29 is what I expected. Looks like this is fixed. Thanks
